core/src/test/java/apoc/text/PhoneticTest.java

```diff
@@ -21,101 +21,11 @@ public void setUp() throws Exception {
         TestUtil.registerProcedure(db, Phonetic.class);
     }
 
-    @Test
-    public void shouldComputeSimpleSoundexEncoding() {
-        testCall(db, "CALL apoc.text.phonetic('HellodearUser!')", (row) ->
-            assertThat(row.get("value"), equalTo("H436"))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('HellodearUser!') as value", (row) ->
-            assertThat(row.get("value"), equalTo("H436"))
-        );
-    }
-
-    @Test
-    public void shouldComputeSimpleSoundexEncodingOfNull() {
-        testCall(db, "CALL apoc.text.phonetic(null)", (row) ->
-                assertThat(row.get("value"), equalTo(null))
-        );
-        testCall(db, "RETURN apoc.text.phonetic(null) as value", (row) ->
-                assertThat(row.get("value"), equalTo(null))
-        );
-    }
-
-    @Test
-    public void shouldComputeEmptySoundexEncodingForTheEmptyString() {
-        testCall(db, "CALL apoc.text.phonetic('')", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('') as value", (row) ->
-                assertThat(row.get("value"), equalTo(""))
-        );
-    }
-
-    @Test
-    public void shouldComputeSoundexEncodingOfManyWords() {
-        testCall(db, "CALL apoc.text.phonetic('Hello, dear User!')", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('Hello, dear User!') as value", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-    }
-
-
-    @Test
-    public void shouldComputeSoundexEncodingOfManyWordsEvenIfTheStringContainsSomeExtraChars() {
-        testCall(db, "CALL apoc.text.phonetic('  ,Hello,  dear User 5!')", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-        testCall(db, "RETURN apoc.text.phonetic('  ,Hello,  dear User 5!') as value", (row) ->
-                assertThat(row.get("value"), equalTo("H400D600U260"))
-        );
-    }
-
```

Comment:
Same type of comment of going from many phonetic tests to now only one. Could be there's a very reasonable reason so please ignore if you feel that's the case.